README.md

```diff
@@ -48,6 +48,7 @@ KubeEdge is composed of the following components:
 ## Usage
 
 * [Prerequisites](./docs/getting-started/usage.md#prerequisites)
```

Comment:
I am wondering if this step is redudant as kubeedge installer will install docker and Kubernetes for us?

Reply:
But what if some one wants to try manually!!? ..We could rename Prerequisities as steps for Manual installation??  Any thoughts???

Reply:
i have changed it to, Manual Installation of Docker,Kubernetes,Mosquitto